src/agentscope/agents/_user_agent.py

```diff
@@ -86,35 +87,37 @@ def reply(
             )
 
             content = raw_input["content"]
-            url = raw_input["url"]
-            kwargs = {}
+            metadata = None
         else:
             # TODO: To avoid order confusion, because `input` print much
             #  quicker than logger.chat
             time.sleep(0.5)
-            content = user_input(timeout=timeout, prefix=self.input_hint)
-            kwargs = {}
+            text_content = user_input(timeout=timeout, prefix=self.input_hint)
+            content = [TextBlock(type="text", text=text_content)]
```

Comment:
It will cause an issue when calling `msg.content` in existing examples for user input

Reply:
fixed